test/server/connection_handler_test.cc

```diff
@@ -1786,10 +1786,9 @@ TEST_F(ConnectionHandlerTest, MatchhIpv4WhenBothIpv4AndIPv6WithIpv4CompatFlag2)
   EXPECT_CALL(*access_log_, log(_, _, _, _));
 }
 
-// This test the case of there is stopped "0.0.0.0" listener, then add
-// "::" listener with ipv4_compat. Ensure the ipv6 listener will take over the
-// ipv4 listener.
-TEST_F(ConnectionHandlerTest, AddIpv4MappedListenerAfterIpv4ListenerStopped) {
```

Comment:
Note for the reviewer: after the fix, this case will be dropped. But it seems fine, it doesn't seem someone needs this case in the real world.